sowasred2012/aws-eks-website
# 1382 EKS page redesign

# Done
Updated Ubuntu AWS EKS image index page per [this brief](https://docs.google.com/document/d/1WMLl_uilhP4TTrwCqw509pR9ZPljUCsy7magujmCg5w/edit#) and [this redesign doc](https://docs.google.com/document/d/1jSFLNu-mS3fM7hR1FZeExInm9P58TKZ_1EDv2elyFpI/edit#heading=h.f99vqlrv5h8u)

Original repo: https://git.launchpad.net/~cloud-images-release-managers/cloud-images/+git/aws-eks-website/

# QA
- Check out this branch
- Open `index.html` in your browser
- Check that the page matches the [redesign doc](https://docs.google.com/document/d/1jSFLNu-mS3fM7hR1FZeExInm9P58TKZ_1EDv2elyFpI/edit#heading=h.f99vqlrv5h8u)

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1382

## Notes
- The lists require date information now, need to find out where to get that from (or whether that's something the CPC team can look at)
